### PR TITLE
Bug 2015422: No conflict checking on snapshot name

### PR DESCRIPTION
**Pull request by avivtur**
**Fixes**:
https://bugzilla.redhat.com/show_bug.cgi?id=2015422
https://issues.redhat.com/browse/OCPBUGSM-37957

**Solution Description**:
Adding snapshot name validation to snapshot modal.
Changing the initial snapshot name creation to avoid conflicts by default with `getSequenceName()`

**Screen shots / Gifs for design review**:

**Before**:

before

**After**:

after

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json [hunk 1]
```diff
@@ -1051,6 +1051,12 @@
   "Interface with this name already exists": "Interface with this name already exists",
   "Network required)": "Network required)",
   "Invalid MAC address format": "Invalid MAC address format",
+  "Snapshot name cannot be empty": "Snapshot name cannot be empty",
+  "Snapshot name name can contain only alphanumeric characters": "Snapshot name name can contain only alphanumeric characters",
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json:1055]: "name" is duplicated

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json [hunk 2]
```diff
@@ -1051,6 +1051,12 @@
...
+  "Snapshot name must start/end with alphanumeric character": "Snapshot name must start/end with alphanumeric character",
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json:1056]: "character" should be plural like the "VM name..." line below.

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json (hunk 1 above)
Comment (avivtur) [reply frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json]: good catch fixed :)

File: frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json (hunk 2 above)
Comment (avivtur) [reply frontend/packages/kubevirt-plugin/locales/en/kubevirt-plugin.json]: good catch fixed :)

**Review by lkladnit — APPROVED** [review]
/lgtm

**Pull request merged**